getting_started/step_by_step/animations.rst

```diff
@@ -30,8 +30,11 @@ The animation editor panel stays visible until manually hidden.
 Creating the animation
 ----------------------
 
-It's time to create a new animation! Press the new animation button and name
-the animation "intro" when the dialog appears.
+It's time to create a new animation! Press the animation button then select
+the new animation button from the menu. name the animation "intro" when
```

Comment:
Too many "buttons". You click the "Animation" button (the name is capitalized" and select "New" from the menu, not another button.

"Name" should be capitalized at the beginning of the third sentence.